metaflow/plugins/argo/argo_workflows.py

```diff
@@ -173,7 +169,8 @@ def __init__(
         self.triggers, self.trigger_options = self._process_triggers()
         self._schedule, self._timezone = self._get_schedule()
 
-        self.kubernetes_labels = self._get_kubernetes_labels()
+        self.shared_kubernetes_labels = self._shared_kubernetes_labels()
```

Comment:
nit - rename both functions to `_base_kubernetes_labels` and `_base_kubernetes_annotations`. Also maybe the variables become - `_base_labels` and `_base_annotations` - at the moment both the variables and functions have the same name which can lead to some confusion.